tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -162,6 +163,10 @@ def bookmark(self, bookmark_title):
     def number_of_columns(self):
         return int(self.marionette.find_element(*self._landing_page_locator).get_attribute('cols'))
 
+    @property
+    def is_homescreen_visible(self):
+        return self.marionette.find_element(*self._homescreen_container_locator)
```

Comment:
find_element will actually raise an exception here.

However, https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/gaia_test.py#L948 is probably the way to go for simplicity's sake.

Just do a simple wait_for_element_displayed against the locator in the test. It'll either raise or succeed. While I don't generally like locators used directly in tests, in this case it keeps it as simple as possible.